<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLoginTable</class>
 <widget class="QDialog" name="CLoginTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>教员管理系统登录</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <item>
      <widget class="QLabel" name="labelLogin">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editUserName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <item>
      <widget class="QLabel" name="labelPassword">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>密  码：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editPassword">
       <property name="inputMask">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>150</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonLogin">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonExit">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonExit</sender>
   <signal>clicked()</signal>
   <receiver>CLoginTable</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
